config/main.py

```diff
@@ -3936,6 +3936,56 @@ def reset(ctx, interface_name):
     cmd = "sudo sfputil reset {}".format(interface_name)
     clicommon.run_command(cmd)
 
+#
+# 'mpls' subgroup ('config interface mpls ...')
+#
+
+@interface.group(cls=clicommon.AbbreviationGroup)
+@click.pass_context
+def mpls(ctx):
+    """Add or remove MPLS"""
+    pass
+
+#
+# 'add' subcommand
+#
+
+@mpls.command()
```

Comment:
why not move mpls into a mpls.py instead of put them in the main.py file?

Reply:
@lguohan You asked this same question 3 months ago.  I will restate my answer from then:  All subgroups, including mpls, of the interface.group are in main.py.  The mpls subgroup is very small and a separate file is overkill.